text/0750-feature-flag-advisor.md

```diff
@@ -0,0 +1,183 @@
+# RFC: Feature Flag Advisor
+
+* Original Author(s): [Vivian Chen](https://quip-amazon.com/GGQ9EAAQ3Jw)
+* Tracking Issue: N/A
+* API Bar Raiser: [Rico Huijbers](https://quip-amazon.com/YPP9EA8Gt6Q)
+
+The feature flag CLI tool will inform users of their current feature flag configuration and allow them to switch any/all of them to a different value.
+
+## Working Backwards
+
+### **Help:** 
```

Comment:
Markdown nit: you don't need to combine `heading` and `bold` styles. Heading alone will suffice.